### PR TITLE
Added information about EvalAI in Readme.

### LINKED ISSUE #10: Add information about EvalAI in README.
Information to add in README file:

- EvalAI Logo
- About EvalAI
- Goal
- Performance comparison

@deshraj @aka-jain @RishabhJain2018 Can I work on this?

### PR DESCRIPTION
**Pull request by Arun-Jain**
Fixes #10 @deshraj @aka-jain @RishabhJain2018 Please review this PR.

### CODE REVIEW

**Review by RishabhJain2018 — CHANGES_REQUESTED** [review]
@Arun-Jain can you please check why images aren't loading? Rest looks good to me!

**Comment by Arun-Jain** [comment]
@RishabhJain2018 Everything is working fine on my side. Can you please review it again?
screenshot from 2018-02-15 12-21-22

File: README.md [hunk 1]
```diff
@@ -42,4 +80,8 @@ You can also use Docker Compose to run all the components of EvalAI-ngx together
 
     ```
     docker-compose -f docker-compose.dev.yml up -d --build
-    ```
\ No newline at end of file
+    ```
+
+## The Team
+
+EvalAI and EvalAI-ngx is currently maintained by [Deshraj Yadav](https://deshraj.github.io), [Akash Jain](http://www.jainakash.in/), [Taranjeet Singh](http://taranjeet.github.io/), [Shiv Baran Singh](http://www.shivbaran.in/) and [Rishabh Jain](https://rishabhjain2018.github.io/). A non-exhaustive list of other major contributors includes: Harsh Agarwal, Prithvijit Chattopadhyay, Devi Parikh and Dhruv Batra.
```
Comment (deshraj) [inline README.md]: Please change it to 
```
EvalAI-ngx is currently maintained by [Akash Jain](http://www.jainakash.in/), [Shiv Baran Singh](http://www.shivbaran.in/), [Rishabh Jain](https://rishabhjain2018.github.io/) and [Deshraj Yadav](https://deshraj.github.io).
```

**Review by deshraj — CHANGES_REQUESTED** [review]
Remove the file `src/assets/images/vqa_leaderboard.png`

**Comment by Arun-Jain** [comment]
@deshraj Updated. Please review the changes.

**Review by deshraj — APPROVED** [review]

**Pull request merged**